bokeh/charts/horizon.py

```diff
@@ -0,0 +1,230 @@
+"""This is the Bokeh charts interface. It gives you a high level API to build
+complex plot is a simple way.
+
+This is the Horizon class which lets you build your Horizon charts just
+passing the arguments to the Chart class and calling the proper functions.
+"""
+
+from six import string_types
+from collections import OrderedDict
+import math
+
+from ._chartobject import ChartObject
+from ..models import ColumnDataSource, Range1d, DataRange1d, FactorRange, HoverTool, CategoricalAxis
+
+#-----------------------------------------------------------------------------
+# Classes and functions
+#-----------------------------------------------------------------------------
+
+
+class Horizon(ChartObject):
+    """This is the Horizon class and it is in charge of plotting
+    Horizon charts in an easy and intuitive way.
+
+    Essentially, we provide a way to ingest the data, separate the data into
+    a number of folds which stack on top of each others.
+    We additionally make calculations for the ranges.
+    And finally add the needed lines taking the references from the source.
+
+    Examples:
+        import datetime
+        from collections import OrderedDict
+        from bokeh.charts import Horizon
+
+        now = datetime.datetime.now()
+        delta = datetime.timedelta(minutes=1)
+        dts = [now + delta*i for i in range(5)]
+        dtss = ['%s'%dt for dt in dts]
+        xyvalues = OrderedDict({'Date': dts})
+        y_python = xyvalues['python'] = [2, 3, 7, 5, 26]
+        y_pypy = xyvalues['pypy'] = [12, 33, 47, 15, 126]
+        y_jython = xyvalues['jython'] = [22, 43, 10, 25, 26]
+
+        ts = Horizon(xyvalues, index='Date', title="horizon",
+                        ylabel='Stock Prices', filename="stocks_ts.html")
+        ts.legend("top_left").show()
+
+    """
+    def __init__(self, values, index=None, title=None, xlabel=None, ylabel=None,
+                 legend=False, xscale="datetime", yscale="linear", width=800,
+                 height=600, tools=True, filename=False, server=False, nb_folds=3,
+                 notebook=False, facet=False, xgrid=False, ygrid=False,
```

Comment:
You should add chart custom attributes consistently to the other charts implementation. Here nb_folds is in between default arguments  and pos_color/neg_color are at the end